frontend/app/src/utils/constants.js

```diff
@@ -76,6 +76,10 @@ export const RunTypes = Object.freeze({
 
 /** Status Codes */
 export const StatusCode = Object.freeze({
+  ON_HOLD: 'ON_HOLD',
+  INITIAL: 'INITIAL',
+  WARNING: 'WARNING',
+  POTENTIAL_DUPLICATE: 'POTENTIAL_DUPLICATE',
   UPLOADINPROGRESS: 'UPLOAD_IN_PROGRESS',
   UPLOADERROR: 'UPLOAD_ERROR',
   PREVALIDATIONCOMPLETED: 'PRE-VALIDATION_COMPLETED',
```

Comment:
could not see this code and many below in the story?

Reply:
Statuses like UPLOADERROR, UPLOADINPROGRESS etc are PHLAT tool specific codes and used when a file getting upload or having an error while upload.

Reply:
then I think we need to separate them.  - row status codes and file status codes